ppr-ui/src/components/mhrRegistration/HomeOwners/HomeOwnerGroups.vue

```diff
@@ -150,14 +150,19 @@ export default defineComponent({
 <style lang="scss" scoped>
 @import '@/assets/styles/theme.scss';
 
+.pad-top {
+  padding-top: 0px;
+}
 #mhr-home-owner-groups::v-deep {
   ul {
     color: $gray7;
     line-height: 24px;
   }
-
   .owner-groups-select {
     width: 200px;
   }
+  .v-application .my-8 {
+    margin-bottom: 0px;
+  }
```

Comment:
This is a little confusing as `my-8` is essentially the same as 32px Top and Bottom. 
Why not change `my-8` to `mt-8` and if the margin-bottom needs adjusting apply a `mb-0` class.

Reply:
fixed.  I like helpers...